packages/design-tokens/src/tokens/color.tokens.json

```
@@ -157,6 +163,51 @@
     "icon-stronger": {
       "value": "#0F172A"
     },
+    "header-background": {
+      "value": "#0B1F9C"
+    },
+    "header-text-inverse": {
+      "value": "#FFFFFF"
+    },
+    "header-decoration": {
+      "value": "#6EE7B7"
+    },
+    "map-icon": {
+      "value": "#4A63FC"
+    },
+    "steps-text": {
+      "value": "#FFFFFF"
+    },
+    "steps-map-background": {
+      "value": "#102EE9"
+    },
+    "footer-text": {
+      "value": "#041162"
+    },
+    "footer-secondary-text": {
```

It seems like in Pluto the convention is type-feature-variant

```suggestion
    "text-footer-secondary": {
```

---

I think it would be good to start with the element like `footer` here